From 35277f419b1dc0f8500d3ab4bceabf6cad05c76d Mon Sep 17 00:00:00 2001
From: Hui Wang <hui.wang@canonical.com>
Date: Tue, 21 Jul 2020 16:51:00 +0800
Subject: [PATCH 39/51] amd/renoir/acp: use the machine driver's name for
 top-level's conf

On the machines with amd renoir audio, the /sys/class/sound/card2/
device/driver links to /sys/bus/platform/drivers/acp_pdm_mach.

Signed-off-by: Hui Wang <hui.wang@canonical.com>
Signed-off-by: Jaroslav Kysela <perex@perex.cz>
---
 ucm2/module/acp_pdm_mach.conf | 1 +
 ucm2/module/snd_acp3x_rn.conf | 1 -
 2 files changed, 1 insertion(+), 1 deletion(-)
 create mode 120000 ucm2/module/acp_pdm_mach.conf
 delete mode 120000 ucm2/module/snd_acp3x_rn.conf

diff --git a/ucm2/module/acp_pdm_mach.conf b/ucm2/module/acp_pdm_mach.conf
new file mode 120000
index 000000000000..b56794a7bc1f
--- /dev/null
+++ b/ucm2/module/acp_pdm_mach.conf
@@ -0,0 +1 @@
+lib/linked.conf
\ No newline at end of file
diff --git a/ucm2/module/snd_acp3x_rn.conf b/ucm2/module/snd_acp3x_rn.conf
deleted file mode 120000
index b56794a7bc1f..000000000000
--- a/ucm2/module/snd_acp3x_rn.conf
+++ /dev/null
@@ -1 +0,0 @@
-lib/linked.conf
\ No newline at end of file
-- 
2.16.4

